core/src/main/java/org/apache/accumulo/core/clientImpl/Tables.java

```diff
@@ -95,28 +92,14 @@ public static TableId getTableId(ClientContext context, String tableName)
     }
   }
 
-  /**
-   * Return the cached ZooCache for provided context. ZooCache is initially created with a watcher
-   * that will clear the TableMap cache for that instance when WatchedEvent occurs.
-   */
   private static ZooCache getZooCache(final ClientContext context) {
     SecurityManager sm = System.getSecurityManager();
     if (sm != null) {
       sm.checkPermission(TABLES_PERMISSION);
     }
 
-    final String uuid = context.getInstanceID();
-
-    try {
-      return instanceToZooCache.get(uuid, () -> {
-        final String zks = context.getZooKeepers();
-        final int timeOut = context.getZooKeepersSessionTimeOut();
-        return new ZooCacheFactory().getZooCache(zks, timeOut,
-            watchedEvent -> instanceToMapCache.invalidate(uuid));
-      });
-    } catch (ExecutionException e) {
-      throw new RuntimeException(e);
-    }
+    return new ZooCacheFactory().getZooCache(context.getZooKeepers(),
```

Comment:
If this is creating a new ZooCache every time it is called, perhaps we shouldn't even be using ZooCache here?

Reply:
No just an instance of the factory class, which keeps a static list of all instances.  The way we use ZooCache, there shouldn't be any need to create an instance of the factory ever.  I am not sure why the class isn't just all static methods.

Reply:
Maybe so we can mock it out in testing?

Reply:
It looks like it.  I think I will change the class to just have static methods since its wasteful to create an object that is only used to get another object.